skada/_ot.py

```diff
@@ -0,0 +1,185 @@
+# Author: Remi Flamary <remi.flamary@polytechnique.edu>
```

Comment:
Should we move OT mapping-based solvers into _ot module as well? (not in this PR). It might be better to keep all of them together to make `POT` into an optional dependency.

Reply:
that's an idea but indeed out of teh csope of this PR